fastrag/readers/T5.py

```diff
@@ -140,8 +163,8 @@ def __call__(self, tokenizer, query: str, documents: List[Document], top_k=None)
         preprocessed_doc = ". ".join([doc.content for doc in documents])
         # preprocessed_doc = [doc.content for doc in documents]
 
-        if self.mode == "Q&A":
-            prompt_with_content = f"question: {query} context: {preprocessed_doc}"
+        if self.mode == "qa":
```

Comment:
Mode name needs to be consistent with https://github.com/IntelLabs/fastRAG/blob/c0eaa589e5d07fe019cf5152e3235eee6b26f578/config/reader/T5.yaml#L9  or define all the possible modes in a central place.